src/pulselistener/pulselistener/mercurial.py

```diff
@@ -159,11 +158,6 @@ def clean(self):
             await asyncio.sleep(1)
 
         # Build and commit try_task_config.json
-        # The build_target_phid variable is stored in the diff
-        # but comes from the Harbormaster query parameters
-        # TODO: use a dedicated variable once we can remove Taskcluster trigger
-        # and simplify this workflow
-        build_target_phid = diff.get('build_target_phid')
```

Comment:
If we don't use it anymore, can we remove the line where we set diff['build_target_phid']?

Reply:
Totally forgot about that :man_facepalming: